### PR TITLE
WPF/ReachFramework:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool

### PR DESCRIPTION
**Pull request by ryalanms**
WPF/ReachFramework:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool.

The RC1 source for these assemblies has 'triple-slash' comments not in the documentation:

*PresentationBuildTasks
PresentationCore
ReachFramework
System.Windows.Controls.Ribbon
System.Windows.Presentation
UIAutomationClient
UIAutomationTypes
WindowsBase*

There is one documentation update PR per assembly. 

(Hide whitespace changes when reviewing files.) 

/cc @dotnet/wpf-developers @carlossanlop

### CODE REVIEW

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentReader.xml [hunk 1]
```diff
@@ -1,353 +1,355 @@
...
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ For more information about the <xref:System.Printing.PrintTicket> part of an XML Paper Specification (XPS) document, see chapter 2 in the XPS specification, which you can obtain at [XPS: Specification and License Downloads](https://www.microsoft.com/whdc/xps/downloads.mspx).  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="N:System.Printing" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Property is not a valid PrintTicket instance.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentReader.xml:241]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceReader.xml [hunk 2]
```diff
@@ -1,179 +1,181 @@
...
+      <ReturnValue>
+        <ReturnType>System.Windows.Xps.Packaging.IXpsFixedDocumentReader</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="documentSource" Type="System.Uri" />
+      </Parameters>
+      <Docs>
+        <param name="documentSource">The URI of the fixed document.</param>
+        <summary>Gets a reader for the <see cref="T:System.Windows.Documents.FixedDocument" /> with the specified uniform resource identifier (URI).</summary>
+        <returns>Returns an interface to an IXpsFixedDocumentReader to read the requested fixed document.</returns>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceReader.xml:94]:
```suggestion
        <returns>Returns an <see cref="T:System.Windows.Xps.Packaging.IXpsFixedDocumentReader"/> instance to read the requested fixed document.</returns>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceReader.xml [hunk 3]
```diff
@@ -1,179 +1,181 @@
...
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ For more information about the <xref:System.Printing.PrintTicket> part of an XML Paper Specification (XPS) sequence, see chapter 2 in the XPS specification, which you can obtain at [XPS: Specification and License Downloads](https://www.microsoft.com/whdc/xps/downloads.mspx).  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="N:System.Printing" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Property is not a valid PrintTicket instance.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceReader.xml:129]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceWriter.xml [hunk 4]
```diff
@@ -1,207 +1,210 @@
...
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentSequenceWriter.AddFixedDocument%2A> method.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="T:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">Current DocumentSequence not completed.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceWriter.xml:67]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">The current document sequence has not completed yet.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceWriter.xml [hunk 5]
```diff
@@ -1,207 +1,210 @@
...
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentSequenceWriter.PrintTicket%2A> property to associate a <xref:System.Printing.PrintTicket> with the <xref:System.Windows.Documents.FixedDocumentSequence>.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Property is not a valid PrintTicket instance.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentSequenceWriter.xml:182]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>
```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml [hunk 6]
```diff
@@ -1,241 +1,246 @@
...
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.AddFixedPage%2A> method.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateadddoccontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateadddoccontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateadddoccontent":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ObjectDisposedException">The FixedDocument has already been disposed</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">FixedPage is not completed.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml:71]:
```suggestion
        <exception cref="T:System.ObjectDisposedException">The fixed document has already been disposed</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PanelOrSequenceAlreadyOpen">The fixed page is not completed.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml [hunk 7]
```diff
@@ -1,241 +1,246 @@
...
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Sets default print options for the <see cref="T:System.Windows.Documents.FixedDocument" />.</summary>
+        <value>The value must be a valid PrintTicket instance.
+             Note:  The PrintTicket can only be assigned to prior to if being committed to the package.  The commit happens when a valid PrintTicket is set and a subsequent flush on the document occurs.</value>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml:199]: Not sure what they mean by "to prior to":

```suggestion
        <value>The value must be a valid <see cref="T:System.Printing.PrintTicket"/> instance.
             Note:  The print ticket can only be assigned to prior to if being committed to the package.  The commit happens when a valid print ticket is set and a subsequent flush on the document occurs.</value>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml [hunk 8]
```diff
@@ -1,241 +1,246 @@
...
+## Examples  
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.IXpsFixedDocumentWriter.PrintTicket%2A> property to associate a <xref:System.Printing.PrintTicket> with a <xref:System.Windows.Documents.FixedDocument>.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">PrintTicket has already been committed.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml:217]: <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml [hunk 9]
```diff
@@ -1,241 +1,246 @@
...
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Property is not a valid PrintTicket instance.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml:218]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedPageReader.xml [hunk 10]
```diff
@@ -1,439 +1,441 @@
...
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ For more information about the <xref:System.Printing.PrintTicket> part of an XML Paper Specification (XPS) page, see chapter 2 in the XPS specification, which you can obtain at [XPS: Specification and License Downloads](https://www.microsoft.com/whdc/xps/downloads.mspx).  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="N:System.Printing" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">part is <see langword="null" />.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">part is <see langword="null" />.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedPageReader.xml:309]: These descriptions are inconsistent compared to the other getters. Shouldn't this be:

```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedPageWriter.xml [hunk 11]
```diff
@@ -1,694 +1,697 @@
...
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Sets the default printing options for the page.</summary>
+        <value>The value must be a valid PrintTicket instance.
+             Note:  The PrintTicket can only be assigned to prior to if being committed to the package.  The commit happens when a valid PrintTicket is set and a subsequent flush on the document occurs.</value>
+        <remarks>To be added.</remarks>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">part is <see langword="null" />.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">part is <see langword="null" />.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedPageWriter.xml:634]: Same. Not consistent. Maybe it should be:

```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```

File: xml/System.Windows.Xps.Packaging/IXpsFixedPageWriter.xml [hunk 12]
```diff
@@ -1,694 +1,697 @@
...
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Sets the default printing options for the page.</summary>
+        <value>The value must be a valid PrintTicket instance.
+             Note:  The PrintTicket can only be assigned to prior to if being committed to the package.  The commit happens when a valid PrintTicket is set and a subsequent flush on the document occurs.</value>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/IXpsFixedPageWriter.xml:631]:
```suggestion
        <value>The value must be a valid <see cref="T:System.Printing.PrintTicket"/> instance.
             Note:  The print ticket can only be assigned to prior to if being committed to the package.  The commit happens when a valid print ticket is set and a subsequent flush on the document occurs.</value>

```

File: xml/System.Windows.Xps.Packaging/XpsDigitalSignature.xml [hunk 13]
```diff
@@ -1,541 +1,549 @@
...
+      <Docs>
+        <param name="certificate">The certificate of authenticity for the signer.</param>
+        <summary>Verifies the document digital signature against a specified X.509 certificate.</summary>
+        <returns>One of the <see cref="T:System.IO.Packaging.VerifyResult" /> enumeration values that specify the result of the verification attempt.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+cannot use this overload with signatures created without embedding their certs
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDigitalSignature.xml:435]: Is this relevant to the final user?

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 14]
```diff
@@ -1,966 +1,984 @@
...
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.XpsDocument.AddFixedDocumentSequence%2A> method to insert content into an <xref:System.Windows.Xps.Packaging.XpsDocument>.  
+  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_Wpf/XpsCreate/CPP/XpsCreate.cpp" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsCreate/CSharp/XpsCreate.cs" id="Snippetxpscreateaddpkgcontent":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsCreate/visualbasic/xpscreate.vb" id="Snippetxpscreateaddpkgcontent":::  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="P:System.Windows.Xps.Packaging.XpsDocument.FixedDocumentSequenceReader" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_AlreadyHasRootSequenceOrDocument">Package already has a root DocumentSequence.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:341]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_AlreadyHasRootSequenceOrDocument">The package already has a root document sequence.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 15]
```diff
@@ -1,966 +1,984 @@
...
+  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/DocumentStructure/CSharp/Window1.xaml.cs" id="Snippetdocstrucfixeddoc":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/DocumentStructure/visualbasic/window1.xaml.vb" id="Snippetdocstrucfixeddoc":::  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="M:System.Windows.Xps.Packaging.XpsDocument.GetFixedDocumentSequence" />
+        <altmember cref="M:System.Windows.Xps.Packaging.XpsDocument.AddFixedDocumentSequence" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotOpenForReading">Package not open for reading.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_InvalidStartingPart">Package starting part is not a valid root.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:567]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotOpenForReading">The package is not open for reading.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_InvalidStartingPart">The package starting part is not a valid root.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 16]
```diff
@@ -1,966 +1,984 @@
...
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Although this class uses a singular name, an <xref:System.Windows.Xps.Packaging.XpsDocument> can be a set (sequence) of one <xref:System.Windows.Documents.FixedDocument> or multiple documents.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PackageUriNull">XpsPakage Uri is <see langword="null" />.  Use XpsDocument constructor that takes Uri parameter.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_InvalidStartingPart">Package starting part is not a valid root.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAFixedDocumentSequence">Part Uri does not corresepond to  a Fixed Document Sequence.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:602]:
```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PackageUriNull">The XpsPakage Uri is <see langword="null" />.  Use the XpsDocument constructor that takes Uri parameter.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_InvalidStartingPart">The package starting part is not a valid root.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAFixedDocumentSequence">The part Uri does not corresepond to  a fixed document Sequence.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 17]
```diff
@@ -1,966 +1,984 @@
...
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="signature" Type="System.Windows.Xps.Packaging.XpsDigitalSignature" />
+      </Parameters>
+      <Docs>
+        <param name="signature">The signature that is deleted.</param>
+        <summary>Deletes a signature from the package.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">signature is <see langword="null" />.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:707]:
```suggestion
        <exception cref="T:System.ArgumentNullException">The signature is <see langword="null" />.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 18]
```diff
@@ -1,966 +1,984 @@
...
+  
+## Examples  
+ The following example shows how to use the <xref:System.Windows.Xps.Packaging.XpsDocument.SignDigitally%2A> method.  
+  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/XpsReadWriteSign/CSharp/SignatureDefinitionsAndSigning.cs" id="Snippetsignxpsdocument":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/XpsReadWriteSign/visualbasic/signaturedefinitionsandsigning.vb" id="Snippetsignxpsdocument":::  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">certificate is <see langword="null" />.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:807]:
```suggestion
        <exception cref="T:System.ArgumentNullException">The certificate is <see langword="null" />.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 19]
```diff
@@ -1,966 +1,984 @@
...
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:858]:
```suggestion
        <exception cref="T:System.ArgumentNullException">The certificate is <see langword="null" />.</exception>

```

File: xml/System.Windows.Xps.Packaging/XpsDocument.xml [hunk 20]
```diff
@@ -1,966 +1,984 @@
...
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Packaging/XpsDocument.xml:914]:
```suggestion
        <exception cref="T:System.ArgumentNullException">The certificate is <see langword="null" />.</exception>

```

File: xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml [hunk 21]
```diff
@@ -1,862 +1,865 @@
...
+        </Attribute>
+      </Attributes>
+      <Parameters>
+        <Parameter Name="xpsPackage" Type="System.Windows.Xps.Packaging.XpsDocument" />
+      </Parameters>
+      <Docs>
+        <param name="xpsPackage">The <see cref="T:System.Windows.Xps.Packaging.XpsDocument" /> that is associated with this <see cref="T:System.Windows.Xps.Serialization.XpsPackagingPolicy" />.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.Xps.Serialization.XpsPackagingPolicy" /> class for a specified <see cref="T:System.Windows.Xps.Packaging.XpsDocument" />.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">reachPackage is NULL.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml:61]: Wrong parameter name.

```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="xpsPackage"/> is <see langword="null"/>.</exception>

```

File: xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml [hunk 22]
```diff
@@ -1,862 +1,865 @@
...
+      <Parameters>
+        <Parameter Name="xpsPackage" Type="System.Windows.Xps.Packaging.XpsDocument" />
+        <Parameter Name="interleavingType" Type="System.Windows.Xps.Packaging.PackageInterleavingOrder" />
+      </Parameters>
+      <Docs>
+        <param name="xpsPackage">The XML Paper Specification (XPS) document that the packaging policy applies to.</param>
+        <param name="interleavingType">The order in which to interleave document elements.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.Xps.Serialization.XpsPackagingPolicy" /> class with a specified <see cref="T:System.Windows.Xps.Packaging.XpsDocument" /> and specified interleave order.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">reachPackage is NULL.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml:88]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="xpsPackage"/> is <see langword="null"/>.</exception>

```

Comment (carlossanlop) [reply xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml:88]: Is it correct to use `null` or is it actually some object called NULL? It has happened before.

File: xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml [hunk 23]
```diff
@@ -1,862 +1,865 @@
...
+        <summary>Stores a specified <see cref="T:System.Printing.PrintTicket" /> as part of the <see cref="T:System.Windows.Xps.Packaging.XpsDocument" />.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method does not validate or modify the specified `printTicket` for a particular <xref:System.Printing.PrintQueue>. If necessary, use the <xref:System.Printing.PrintQueue.MergeAndValidatePrintTicket%2A> method to create a <xref:System.Printing.PrintTicket> that is specific to the <xref:System.Printing.PrintQueue> and is valid for a specified printer.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">printTicket is NULL.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml:540]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="printTicket"/> is <see langword="null"/></exception>

```

File: xml/System.Windows.Xps.Serialization/XpsSerializationManager.xml [hunk 24]
```diff
@@ -1,304 +1,306 @@
...
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="serializedObject" Type="System.Object" />
+      </Parameters>
+      <Docs>
+        <param name="serializedObject">The XAML serialized object to save.</param>
+        <summary>Saves a specified XAML serialized object to the document package.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">serializedObject is NULL.</exception>
+        <exception cref="T:System.Windows.Xps.XpsSerializationException">serializedObject is not a supported type.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Serialization/XpsSerializationManager.xml:150]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="serializedObject"/> is <see langword="null"/></exception>
        <exception cref="T:System.Windows.Xps.XpsSerializationException"><paramref name="serializedObject"/> is not a supported type.</exception>

```

File: xml/System.Windows.Xps.Serialization/XpsSerializationManagerAsync.xml [hunk 25]
```diff
@@ -1,167 +1,169 @@
...
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="serializedObject" Type="System.Object" />
+      </Parameters>
+      <Docs>
+        <param name="serializedObject">The object that is saved.</param>
+        <summary>Saves a specified serialized object to an XML Paper Specification (XPS) package.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">serializedObject is NULL.</exception>
+        <exception cref="T:System.Windows.Xps.XpsSerializationException">serializedObject is not a supported type.</exception>
```
Comment (carlossanlop) [inline xml/System.Windows.Xps.Serialization/XpsSerializationManagerAsync.xml:134]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="serializedObject"/> is <see langword="null"/></exception>
        <exception cref="T:System.Windows.Xps.XpsSerializationException"><paramref name="serializedObject"/> is not a supported type.</exception>

```

**Comment by gewarren** [comment]
Close/reopen for fresh build.

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentReader.xml (hunk 1 above)
Comment (ryalanms) [reply xml/System.Windows.Xps.Packaging/IXpsFixedDocumentReader.xml:241]: I think we should keep the name of the class instead of translating to a friendly name.   Reactive if you disagree.  Thanks.

File: xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml (hunk 6 above)
Comment (ryalanms) [reply xml/System.Windows.Xps.Packaging/IXpsFixedDocumentWriter.xml:71]: FixedDocument and FixedPage are classes.

**Comment by gewarren** [comment]
Close/reopen for fresh build.

**Comment by gewarren** [comment]
Closing due to age and number of conflicts.
